app/src/main/java/com/android/streetworkapp/model/moderation/PerspectiveAPIModels.kt

```diff
@@ -0,0 +1,107 @@
+package com.android.streetworkapp.model.moderation
+
+import kotlinx.serialization.Serializable
+
+// All the definitions below are used to interact with Perspective API
+
+/** Default values we can use as thresholds for our TextModerationTags */
+object PerspectiveAPIThresholds {
+  val DEFAULT_THRESHOLD_VALUES = TextModerationTags.entries.associateWith { 0.2 }
+}
+
+/** All the tags we want to fetch from the API */
+enum class TextModerationTags {
+  TOXICITY,
+  INSULT,
+  THREAT,
+  PROFANITY
+}
```

Comment:
Question: Is there a way to identify https links? I think avoiding advertising/harmful links could also be useful.

Reply:
Nope, but I'm pretty sure I could implement something myself that does that. I'll see what I can do.

Here's all the tags we can use from the API if you're interrested: https://developers.perspectiveapi.com/s/about-the-api-attributes-and-languages?language=en_US
There's a thing for spam that could maybe work for links? But it's only for english

Reply:
I've looked around, I could probably do something with Regex but there's several drawbacks: Either I miss some links, or I get them all and there's a bunch of false positives. As links aren't clickable by default in Texts in Android (from what I've seen), I don't think it's worth the time. We could put a little disclaimer the first time a user opens an event/logs in though

Reply:
I would argue that "get all links" is perhaps the best solution, as there is probably, in our app, no good reason to let a user post any link. But I think we are going outside the scope of this PR.

Reply:
Yeah that's something for another pr. I think it's complicated to get all links without ruining a part of the user experience. 
If you look at this page for example: https://fr.wikipedia.org/wiki/Liste_des_domaines_Internet_de_premier_niveau
I can easily see happening that a user types '${word}.${some domain name}' by mistake because he forgot to add a space after ponctuation.